plugin/db/driver.go

```diff
@@ -375,6 +375,9 @@ type Driver interface {
 	// limit is the maximum row count returned. No limit enforced if limit <= 0
 	Query(ctx context.Context, statement string, limit int) ([]interface{}, error)
 
+	// QueryString returns the query where clause string for the query parameters.
+	QueryString(*QueryParams) string
```

Comment:
QueryString() probably shouldn't be part of Driver interface.

I made https://github.com/bytebase/bytebase/pull/526 to fix this.